### PR TITLE
Allow creating a new wallet from the `Enter Password` screen

### PR DESCRIPTION
**Pull request by CassioMG**
Closes https://github.com/stellar/freighter/issues/1472

This PR adds a `Create a wallet` button at the bottom of the `Enter Password` screen so that users are able to create a new wallet in case they don't have the password or the seed phrase saved to import an existing wallet.

Screenshot 2024-11-12 at 15 45 26

https://github.com/user-attachments/assets/2373d1af-2bd2-4e02-a6f4-583446d80eba

### CODE REVIEW

File: extension/src/popup/locales/en/translation.json [hunk 1]
```diff
@@ -6,6 +6,7 @@
   "A Stellar wallet for every website": "A Stellar wallet for every website",
   "About": "About",
   "Account": "Account",
+  "Account details": "Account details",
```
Comment (piyalbasu) [inline extension/src/popup/locales/en/translation.json:9]: Sorry you're getting so many of these unrelated translation updates in your PR's. There's a github action that's supposed to add these automatically when devs open their PR's, but apparently it stopped working? I'll take a look.

In the future, you should only have added translations for any strings that you updated in your PR

**Review by piyalbasu — APPROVED** [review]

**Review by aristidesstaffieri — APPROVED** [review]

**Comment by JakeUrban** [comment]
The title is [DO NOT MERGE], should this be a draft PR? Or does the title need updating?

**Comment by CassioMG** [comment]
> The title is [DO NOT MERGE], should this be a draft PR? Or does the title need updating?

@JakeUrban I was following the same pattern from this other 2 PRs (#1631, #1630). I think making it a Draft helps making sure it won't be merged, but on the other hand it may not be very clear for other devs if a Draft PR is something they should be reviewing already or not.

Maybe we could convert those to Draft PRs and add a [READY FOR REVIEW] tag + tagging reviewers once we think something is ready for review?

@JakeUrban @aristidesstaffieri @aristidesstaffieri what do you guys think about it?

**Comment by CassioMG** [comment]
> > The title is [DO NOT MERGE], should this be a draft PR? Or does the title need updating?
> 
> @JakeUrban I was following the same pattern from this other 2 PRs (#1631, #1630). I think making it a Draft helps making sure it won't be merged, but on the other hand it may not be very clear for other devs if a Draft PR is something they should be reviewing already or not.
> 
> Maybe we could convert those to Draft PRs and add a [READY FOR REVIEW] tag + tagging reviewers once we think something is ready for review?
> 
> @JakeUrban @aristidesstaffieri @aristidesstaffieri what do you guys think about it?

Just adding a little bit more context. We are holding those PRs because we shouldn't be including new changes to `v5.25.0` at this point. So we are waiting for it to be released to start merging stuff on the new release branch (as master should only receive released changes given our current gitflow approach).

**Comment by CassioMG** [comment]
@aristidesstaffieri @piyalbasu FYI, I've rebased this PR on `v5.25.1` since it's the next release.
Do we want to merge this PR to `v5.25.1` or should it wait for `v5.26.0` to be merged?

**Pull request merged**